pnpm-workspace.yaml

```diff
@@ -0,0 +1,3 @@
+packages:
+  - mw-webapp
+  - masters-way
```

Comment:
somethisng wrong here, we don't have package masters-way as I remember, but we have mw-server instead

Reply:
I can remove masters-way module, i added the root module but i didn't add server because it doesn't have package.json yet

Reply:
done

Reply:
does all modules shuold have package.json?